examples/op_examples/schema.py

```diff
@@ -0,0 +1,18 @@
+from datasets import load_dataset
+
+import lotus
+from lotus.models import LM
+
+lm = LM(model="ollama/llama3.1")
+
+lotus.settings.configure(lm=lm)
+
+dataset = load_dataset("CShorten/ML-ArXiv-Papers", split="train")
+df = dataset.to_pandas().head(3)
+
+columns = ["problem", "dataset", "results"]
+col_descriptions = ["Description of the problem", "What dataset is used", "What results are obtained"]
+
+user_instruction = "{abstract}"
+new_df = df.sem_schema(user_instruction, columns, col_descriptions)
```

Comment:
```
Traceback (most recent call last):
  File "/Users/sidjha/Workspace/Research/lotus/examples/op_examples/schema.py", line 17, in <module>
    new_df = df.sem_schema(user_instruction, columns, col_descriptions)
  File "/Users/sidjha/miniconda3/envs/lotus/lib/python3.10/site-packages/pandas/core/generic.py", line 6299, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'sem_schema'
```

I think you need to add `sem_schema` to the `__init__.py` [here](https://github.com/TAG-Research/lotus/blob/main/lotus/__init__.py)